onedal/utils/_dpep_helpers.py

```diff
@@ -0,0 +1,50 @@
+# ==============================================================================
+# Copyright 2024 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+"""Utilities for Data Parallel Extensions libs, such as DPNP, DPCtl"""
+
+from daal4py.sklearn._utils import _package_check_version
+
+
+def is_dpctl_available(version=None):
```

Comment:
These functions need caching just in case they will be used more extensively in future.